pkg/test/framework/components/prometheus/prometheus.go

```diff
@@ -27,29 +27,31 @@ type Instance interface {
 
 	// API Returns the core Prometheus APIs.
 	API() v1.API
+	APIForCluster(cluster resource.Cluster) v1.API
 
 	// WaitForQuiesce runs the provided query periodically until the result gets stable.
 	WaitForQuiesce(fmt string, args ...interface{}) (prom.Value, error)
 	WaitForQuiesceOrFail(t test.Failer, fmt string, args ...interface{}) prom.Value
+	WaitForQuiesceForCluster(cluster resource.Cluster, fmt string, args ...interface{}) (prom.Value, error)
```

Comment:
It may be an easier API to use, or more clear which cluster we're reaching if rather than each method having a `ForCluster` variant, we used a `PromForCluster` to get this interface.

Reply:
This is more of a cosmetic thing and isn't really necessary

Reply:
Okay.. will take a look at it later then...